pkg/meta/base.go

```diff
@@ -242,6 +242,38 @@ func (m *baseMeta) checkRoot(inode Ino) Ino {
 	}
 }
 
+func (m *baseMeta) batchCalcDirStat(ctx Context, stats map[Ino]*dirStat) error {
+	var eg errgroup.Group
+	for i, s := range stats {
+		ino, stat := i, s
+		eg.Go(func() error {
```

Comment:
They are independent, so we should not cancel others if one of them fail

Reply:
Also, we cant to update the stats right after the calculation, don't wait for other large directories. So it's better to move the parallization out of calculation

Reply:
Done